### PR TITLE
docs: format docs and typos fix

### PR DESCRIPTION
**Pull request by Mister-Hope**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [x] Docs

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

This PR formate docs, such as:

- changing mixing `` ```js ``, `` ```javascript `` to `` ```js ``
- changing mixing list charactor `* aaa` `- aaa` to `- aaa`
- remove duplicate lines
- add code block on some words
- space English words and Chinese words
- typo fix

### CODE REVIEW

File: packages/docs/docs/theme/default-theme-config.md [hunk 1]
```diff
@@ -19,14 +19,15 @@ tagline: Hero subtitle
...
-- title: Vue-Powered
-  details: Enjoy the dev experience of Vue + webpack, use Vue components in markdown, and develop custom themes with Vue.
-- title: Performant
-  details: VuePress generates pre-rendered static HTML for each page, and runs as an SPA once a page is loaded.
+  - title: Simplicity First
+    details: Minimal setup with markdown-centered project structure helps you focus on writing.
+  - title: Vue-Powered
+    details: Enjoy the dev experience of Vue + webpack, use Vue components in markdown, and develop custom themes with Vue.
+  - title: Performant
+    details: VuePress generates pre-rendered static HTML for each page, and runs as an SPA once a page is loaded.
```
Comment (Mister-Hope) [inline packages/docs/docs/theme/default-theme-config.md:27]: I supposted the linter in ci can not deal with Frontmatter part well.

I think this is the correct syntax, and no erros should be thrown

**Comment by bencodezen** [comment]
Hi @Mister-Hope! Thanks for the PR on this one. This PR introduces a lot of style changes across the file that should probably be enforced at the CI level as you mentioned. Since the team is still working on establishing its own coding standards, we'll have to close this one for now. Thank you!

**Comment by bencodezen** [comment]
Also, forgot to mention this, in the future, breaking up the PR into smaller chunks would make it easier to review and approve / request changes. If you feel there are parts that are worth discussing, please submit another PR with only those changes. Thank you!

**Comment by Mister-Hope** [comment]
I will reopen one only with typo change.

**Comment by Mister-Hope** [comment]
Besides, the pr which I request for dark mode, when can you review it? A month has passed.

Also, I have about 8 other opening PR

**Comment by bencodezen** [comment]
@Mister-Hope Sorry for the delay on this. We will do our best to get to the open PRs. Thanks for your patience!

**Comment by Mister-Hope** [comment]
> Hi @Mister-Hope! Thanks for the PR on this one. This PR introduces a lot of style changes across the file that should probably be enforced at the CI level as you mentioned. Since the team is still working on establishing its own coding standards, we'll have to close this one for now. Thank you!

BTW, when do you plan to establish it? I hope the tool can have full check of all the frontmatter types.

No offense, but due to the efficiency in the last half year, is it still in your mind? ༼ つ ◕_◕ ༽つ

**Comment by bencodezen** [comment]
@Mister-Hope Currently the team is pre-occupied with work for Vue 3, but I definitely plan on increasing the efficiency of the team once the Vue 3 work has wrapped up.

**Pull request closed without merging**